casper/src/main/resources/RevVault.rho

```diff
@@ -175,100 +219,139 @@ in {
               )
             }
           } |
-          // Supplies balance, transfer, and setLog methods on revVault.
-          contract _newVault(revVault, @ownRevAddress, purse) = {
-            new logStore in {
-              logStore!(Nil) |
-              contract revVault(@"balance", ret) = {
-                purse!("getBalance", *ret)
-              } |
-              // Transfers from revVault to toRevAddress.
-              contract revVault(@"transfer", @targetAddress, @amount, authKey, ret) = {
```

Comment:
@tgrospic 
Conflicts here because the renaming `targetAddress` here. I used Issac's name `fromRevAddress` now. Let me know if you have questions.